src/motion_input_shaping/shaping_comparison_test.py

```diff
@@ -9,13 +9,14 @@
 
 import time
 import sys
-from matplotlib import pyplot as plt  # type: ignore
+from matplotlib import pyplot as plt
 from zaber_motion import Units
 from zaber_motion.ascii import Connection
 from shaped_axis import ShapedAxis  # Zaber axis with input shaping
```

Comment:
Comment probably unncessary

Reply:
I have removed all the comments for these imports. It is all described in the docstrings anyways